rabbitmq_measurement_exchanger.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"github.com/streadway/amqp"
+	"log"
+	"strconv"
+)
+
+type rabbitMQMeasurementExchanger struct {
+	measurements *amqp.Channel
+}
+
+func newRabbitMQMeasurementExchanger(url string) (*rabbitMQMeasurementExchanger, error) {
+	conn, err := amqp.Dial(url)
+	if err != nil {
+		return nil, err
+	}
+
+	measurements, err := conn.Channel()
+	if err != nil {
+		return nil, err
+	}
+
+	err = measurements.ExchangeDeclare(
+		"measurements",
```

Comment:
Let's move exchange name to env variable. We usually keep it in env cause it's usually shared between multiple services and it's easier to configure all of them properly this way.